### PR TITLE
docs: add information to the workspace docs on how to use extends

### PR DESCRIPTION
**Pull request by CoolGoose**
### Description

This should make it a bit clearer on how to reuse a base config file in our workspaces since implicitly they don't inherit from anything.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [x] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

**Review by sheremet-va — CHANGES_REQUESTED** [review]
Please, lint your edits according to the project eslint config

**Comment by CoolGoose** [comment]
@sheremet-va done senpai, didn't realize the CI broke 😅

File: docs/guide/workspace.md [hunk 1]
```diff
@@ -172,6 +172,30 @@ export default mergeConfig(
 ```
 :::
 
+At the `defineWorkspace` level you can use the `extends` option to inherit from your root-level config.
```
Comment (sheremet-va) [inline docs/guide/workspace.md:175]: Just noticed, but how is this different from 

https://github.com/CoolGoose/vitest/blob/266330c3dee2ff55ec1fc1aad1fdc03aff2a9b6b/docs/guide/workspace.md?plain=1#L62

Comment (CoolGoose) [reply docs/guide/workspace.md:175]: It's not fully different, I wrote it mostly in the context of 'please make sure you understand there's no base config automatically extended when you're doing workspaces' a bit localized to the section that talks about Configuration specifically.

Looking back on how I wrote it it's a bit clunky indeed.

The problem is that conceptually, I think it was more of a case of expecting the workspaces to automatically extend the vitest.config.ts (especially in the context that not all options are available in the workspace config, so you'll need to use extends). and also the difference of `mergeConfig` for vitest and `.extends` in the workspace 😅 

Happy to just close this if you think it's just a bit more confusing.

Comment (sheremet-va) [reply docs/guide/workspace.md:175]: I see, I guess it makes sense to also duplicate an example with `extends` here alongside `mergeConfig` as an alternative

Comment (sheremet-va) [inline docs/guide/workspace.md:175]:
```suggestion
At the `defineWorkspace` level you can also use the `extends` option instead to inherit from your root-level config.
```

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**